lib/web_ui/test/engine/platform_dispatcher/view_focus_binding_test.dart

```diff
@@ -68,28 +68,6 @@ void testMain() {
       expect(view2.dom.rootElement.getAttribute('tabindex'), '0');
     });
 
-    test('never marks the views as focusable with semantincs enabled', () async {
```

Comment:
Should we also _add_ some tests? :)

Reply:
Well, there's tests already (this file is just showing the removed test, but there's others to verify the rest of the functionality!)